{"uuid": "f53e7626-02b2-466b-9de8-3d01f65ec058", "children": ["72ff71be-1e1a-49a9-840f-19906a8aa12f"], "befores": [{"name": "initialization_driver", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))\n", "trace": "  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\src\\PageObjectModel\\Tests\\conftest.py\", line 50, in initialization_driver\n    web_driver = webdriver.Remote(\n                 ^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 344, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 290, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 311, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 118, in request\n    return self.request_encode_body(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 217, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\poolmanager.py\", line 433, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 844, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\util\\retry.py\", line 470, in increment\n    raise reraise(type(error), error, _stacktrace)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\util\\util.py\", line 38, in reraise\n    raise value.with_traceback(tb)\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 790, in urlopen\n    response = self._make_request(\n               ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 536, in _make_request\n    response = conn.getresponse()\n               ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\urllib3\\connection.py\", line 454, in getresponse\n    httplib_response = super().getresponse()\n                       ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\http\\client.py\", line 1375, in getresponse\n    response.begin()\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\http\\client.py\", line 318, in begin\n    version, status, reason = self._read_status()\n                              ^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\http\\client.py\", line 279, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\socket.py\", line 706, in readinto\n    return self._sock.recv_into(b)\n           ^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1686268489270, "stop": 1686268571397}], "start": 1686268489270, "stop": 1686268571944}